packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/LoginCardSocialLink.js

```diff
@@ -13,11 +14,28 @@ const LoginCardSocialLink = ({ link }) =>
   );
 
 LoginCardSocialLink.propTypes = {
-  link: PropTypes.shape({ ...Link.propTypes })
+  /** the link element props. */
+  link: PropTypes.shape({
+    ...Link.propTypes,
+    /** The image source */
+    src: PropTypes.string.isRequired,
+    /** The image alt description */
+    alt: PropTypes.string,
+    /** The link text */
+    text: PropTypes.string,
+    /** Callback to trigger when clicking the checkbox */
+    onClick: PropTypes.func
+  })
 };
 
 LoginCardSocialLink.defaultProps = {
-  link: { ...Link.defaultProps, onClick: e => e.preventDefault() }
+  link: {
+    ...Link.defaultProps,
+    src: null,
+    alt: null,
+    text: null,
+    onClick: noop
```

Comment:
heads up, i think `onClick` is already part of the default props... be curious to know why it wasn't handled that way before, maybe an oversight